File: Java/BubbleSort.java
public class BubbleSort {
    public static void main(String[] args) {
        int[] arr = {95,9,45,12,74,56,54,41};

        bubbleSort(arr);
        for (int i : arr) {
            System.out.print(i + " ");
        }
    }

    static void bubbleSort(int[] arr){
        for (int i = 0; i < arr.length-1; i++) {
            int flag=0,temp;
            for (int j = 0; j < arr.length-1-i; j++) {
                if (arr[j]>arr[j+1]){
                    temp=arr[j];
                    arr[j]=arr[j+1];
                    arr[j+1]=temp;
                    flag=1;
                }
            }
            if (flag==0) break;
        }
    }
}

File: JavaScript/Bubble_Sort.js
function bubbleSort(arr){
    
    var i, j;
    var len = arr.length;
      
    var isSwapped = false;
      
    for(i =0; i < len; i++){
        
      isSwapped = false;
        
      for(j = 0; j < len; j++){
          if(arr[j] > arr[j + 1]){
            var temp = arr[j]
            arr[j] = arr[j+1];
            arr[j+1] = temp;
            isSwapped = true;
          }
      }
        
      // IF no two elements were swapped by inner loop, then break 
        
      if(!isSwapped){
        break;
      }
    }
      
    // Print the array
    console.log(arr)
  }
    
    
  var arr = [243, 45, 23, 356, 3, 5346, 35, 5];
    
  // calling the bubbleSort Function
  bubbleSort(arr)